src/commands/deploy/genezio.ts

```diff
@@ -675,7 +676,26 @@ export async function deployFrontend(
 
     try {
         await doAdaptiveLogAction(`Building frontend ${index + 1}`, async () => {
-            await runScript(frontend.scripts?.build, frontend.path);
+            debugLogger.debug(
+                `Running build script for frontend ${index + 1}`,
+                frontend.scripts?.build,
+            );
```

Comment:
Is this relevant to be printed before expansion? I would print the script content after expansion.

Reply:
Nope, I forgot it here, thanks!